cmd/util/cmd/remove-execution-fork/cmd/execution-fork.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/cmd/util/cmd/common"
+	"github.com/onflow/flow-go/storage"
+	"github.com/onflow/flow-go/storage/badger/operation"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "remove-execution-fork",
+	Short: "Remove execution fork for conflicting results. Useful for resuming the sealing.",
+	Run:   run,
+}
+
+func init() {
+	rootCmd.AddCommand(Cmd)
+}
+
+func run(*cobra.Command, []string) {
+	log.Info().
+		Str("datadir", flagDatadir).
+		Msg("flags")
+
+	db := common.InitStorage(flagDatadir)
+	defer db.Close()
+
+	err := db.Update(operation.RemoveExecutionForkEvidence())
+
+	// for testing purpose
+	// expectedSeals := unittest.IncorporatedResultSeal.Fixtures(2)
+	// err := db.Update(operation.InsertExecutionForkEvidence(expectedSeals))
+
+	if err == storage.ErrNotFound {
+		log.Info().Msg("no execution fork was found, exit")
+		return
+	}
+
+	if err != nil {
+		log.Fatal().Err(err).Msg("could not remove execution fork")
```

Comment:
JFYI `db.Close` from line 27 won't be closed when we log a fatal error since it calls `os.Exit` which won't run deferred functions. I think this would result in the database staying locked after the command returns.

Reply:
I have tested. Even if `Close()` is not called, the database is not locked. I'm able to insert the evidence with the test code, and force to run into a fatal, and then run again with out fatal, and managed to remove the fork evidence.

I think badger has handed that crash case already.